src/map.cpp

```diff
@@ -3382,6 +3382,10 @@ void map::shoot( const tripoint &p, int &dam,
         add_field(p, fd_fire, rng(1, 2), 0 );
     }
 
+    if (ammo_effects.count("STREAM_GAS") && !one_in(3)) {
```

Comment:
Something more descriptive would be better. Something like "FUNGICIDAL_STREAM" or "FUNGICIDAL_TRAIL".